decision_making.md

```diff
@@ -0,0 +1,42 @@
+# Decision Making Guide
+
+This document describes how Jupyter’s governing bodies (Board of Directors, Software Steering Council, Software Projects, non-software Working Groups) make decisions. Because individuals often work across multiple decision making groups, all Jupyter governing bodies are expected to follow this approach.
+
+This document provides guidelines but is not meant to be a comprehensive procedure manual. We seek to honor the principles of collaboration, inclusive participation, and responsive decision making. If a situation arises that is not explicitly covered by the guidelines in this document, we encourage all teams to find a path forward with decisions, filling in the gaps in formal decision making procedure with adherence to these principles.
+
+## Formal aspects of decision making
+
+The decision making process is intended to balance broad participation of stakeholders with agility. While the size of decision making bodies may vary (e.g. software projects can range from a handful to dozens of decision makers), all need to find this balance.
+
+All official Jupyter projects, working groups, governing bodies, and official initiatives must have a clearly defined, and documented decision making body. Formally, these decision making bodies make decisions using consensus seeking with an option to call a vote to move the decision forward.
+
+Depending on the governing body, decisions may be proposed by members of the decision making body, or by a broader group of stakeholders/contributors.
+
+Decision making, even when it begins with simple informal discussion, starts with consensus seeking. The goal of this discussion phase is to refine the proposal, consider alternatives, weigh trade-offs, and attempt to find informal consensus. The legitimacy of the consensus seeking process is predicated on all stakeholders having their voice heard; teams must be proactive in providing opportunity for all relevant stakeholders to provide input. If the decision making body arrives at informal consensus, they may immediately move to document and enact the decision. This is the consensus seeking phase.
+
+At any time during the consensus seeking phase, any member of the decision making body can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current form and notify the entire decision making body. After the proposal is seconded by another member of the decision making body, members have 7 days to vote. The governing body may consider longer voting periods as necessary for special circumstances, or shorter periods only if all voting members are present. The result will be determined with no quorum by a simple majority for binary decisions and ranked choice for multiclass decisions (one among many, or many among many). The sponsor may update the proposal at any point during the voting period, in which case, the voting period will be reset.
```

Comment:
I feel like there are certain kinds of decisions where you really do want quorum (e.g. a board of directors, a project-wide decision on a JEP, etc) and others where you probably don't need it. Enforcing quorum/no-quorum across all decisions will likely feel inadequate for one or the other type of decision unless we differentiate the two somehow.

Perhaps we can identify an explicit subset of decision-making bodies (e.g. board of directors), or types of decisions (e.g. project-wide JEPs) that require quorum, and for others it is left to the project to decide?

Reply:
Governance decisions (BoD and JEPs) benefit from having a quorum. Without quorum and minutes, there is no way to measure the effectiveness of transparency or adequately make sure that folks are aware of a vote.

Decisions that do not need quorum are those that are more along the lines of day-to-day operations of the project.

Reply:
I like this idea of quorum/no-quorum for well-defined subsets.

Reply:
Can we be explicit that BoD and SSC votes should have a quorum? Day-to-day operations in the subproject could be exempt from that. Lack of quorum in BoD and SSC votes leaves me concerned about the governance.